### PR TITLE
Create parking transaction history DAG

### PR DESCRIPTION
**Pull request by johnclary**
Resolves https://github.com/cityofaustin/atd-data-tech/issues/7890 by deploying https://github.com/cityofaustin/atd-parking-data/tree/production/meters

### CODE REVIEW

**Comment by johnclary** [comment]
@sergiogcx @chiaberry i'm deploying this 🤞

**Comment by johnclary** [comment]
Derp—forgot update images JSON. Done [here](https://github.com/cityofaustin/atd-airflow/pull/78).

**Comment by chiaberry** [comment]
parking

**Review by sergiogcx — COMMENTED** [review]
👍

File: dags/atd_parking_data_meters_txn_history.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
+from airflow.models import DAG
+from airflow.models import Variable
+from airflow.operators.docker_operator import DockerOperator
+from _slack_operators import *
+
+default_args = {
+    "owner": "airflow",
+    "description": "Publish sign work order specifications to Postgres, AGOL",
```
Comment (chiaberry) [inline dags/atd_parking_data_meters_txn_history.py:9]: just fyi 👀

**Pull request merged**